lnclient/lnd/lnd.go

```diff
@@ -313,34 +313,54 @@ func (svc *LNDService) LookupInvoice(ctx context.Context, paymentHash string) (t
 	return transaction, nil
 }
 
+func (svc *LNDService) getPaymentResult(stream routerrpc.Router_SendPaymentV2Client) (*lnrpc.Payment, error) {
+	for {
+		payment, err := stream.Recv()
+		if err != nil {
+			return nil, err
+		}
+
+		if payment.Status != lnrpc.Payment_IN_FLIGHT {
+			return payment, nil
+		}
+	}
+}
+
 func (svc *LNDService) SendPaymentSync(ctx context.Context, payReq string, amount *uint64) (*lnclient.PayInvoiceResponse, error) {
-	sendRequest := &lnrpc.SendRequest{PaymentRequest: payReq}
+	const MAX_PARTIAL_PAYMENTS = 21
+	const SEND_PAYMENT_TIMEOUT = 60
```

Comment:
note: LDK is set to 50 seconds currently, since the JS SDK at least uses a 60 second timeout for NWC requests (although all this should be re-reviewed)

Reply:
I took 60 from here: https://github.com/getAlby/hub/blob/master/lnclient/ldk/ldk.go#L465, should this be changed?

Reply:
True, we should probably set that to 50 seconds too. Could you update it?

Reply:
Done!